.github/bug_report.md

```diff
@@ -0,0 +1,26 @@
+### **Describe the bug**
+A clear and concise description of what the bug is.
+
+### **To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+### **Expected behavior**
+A clear and concise description of what you expected to happen.
+
+### **Screenshots**
+If applicable, add screenshots or screen captures to help explain your problem.
+
+### **Additional context**
+Add any other context about the problem here.
+
+<hr/>
+
+<!--- Do not remove or change this in the issue description. Only update the details above this. --->
+**Note:**
+* If you want to work on an issue, you should check if it has already been assigned to anyone. **If the issue is free** you can comment `/assign` to get the issue assigned to you.
+* If you are raising a new issue and want to work on it then also you should comment `/assign` under the issue to get it auto assigned.
+* Please **refrain from** adding labels to your issue/pull-request on your own. It is the job of the Project Admin and the Mentors to review your issue/pull-request and add labels accordingly.
```

Comment:
I believe `/assign` is a function of PROW which our project doesn't use. Can we change this to state that something like "if you would like to be assigned to it please comment on the issue".